rpcd/playbooks/roles/rpc_maas/tasks/raxmon_agent_install.yml

```diff
@@ -39,12 +48,20 @@
     mode: "0600"
 
 - name: Verify raxmon-entities-list and MaaS API authentication does not fail
-  command: "raxmon-entities-list"
+  shell: |
+    {% if maas_venv_enabled | bool %}
```

Comment:
@alextricity25 except it's in nine places. If it were two or three, I wouldn't really care that much. That's 9 places to update if we change something about the variable names.

Anyway, this isn't a blocker. We can clean it up in a follow-up PR later.